src/Pelagos/Bundle/AppBundle/Resources/views/Email/user.dataset-created.email.twig

```diff
@@ -7,7 +7,7 @@
     <br>
     The Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) has received your dataset submission {{ datasetSubmission.dataset.udi }}.
     The dataset landing page is <a href="{{ url('pelagos_app_ui_dataland_default', { 'udi': datasetSubmission.dataset.udi}) }}">{{url('pelagos_app_ui_dataland_default', { 'udi': datasetSubmission.dataset.udi})}}</a>.<br>
-    This dataset has been assigned the doi: {{datasetSubmission.dataset.doi.doi}}.<br><br>
+    This dataset has been assigned the doi: {{datasetSubmission.dataset.doi|default}} .<br><br>
     The dataset package will be reviewed by GRIIDC.<br><br>
     If you have any questions regarding this dataset submission please email griidc@gomri.org.<br>
     <br>
```

Comment:
Line 22 breaks too. `This dataset has been assigned the doi: {{datasetSubmission.dataset.doi.doi}}.`